README.md

```diff
@@ -3,6 +3,23 @@ eLife Profiles
 
 [![Build Status](https://ci--alfred.elifesciences.org/buildStatus/icon?job=test-profiles)](https://ci--alfred.elifesciences.org/job/test-profiles/) [![Coverage Status](https://coveralls.io/repos/github/elifesciences/profiles/badge.svg?branch=develop)](https://coveralls.io/github/elifesciences/profiles?branch=develop)
 
+Profiles is a service used by eLife Sciences Publications, Ltd. to create and 
+store eLife user profiles. Currently, eLife uses [ORCID](https://orcid.org/) for 
+authentication.
+
+When a user account is created or updated on ORCID, the new account information is sent to 
```

Comment:
Incorrect: we setup one webhook per user when we create the account, so the flow is:
- when a user authenticates through profiles using ORCID OAuth, a user account is created; or updated if existing
- when a user account is updated on ORCID, the webhook notifies this service to update the existing user account

The main point being we create a user only through voluntary authentication, with a very small subset of the ORCID population that happens to use elifesciences.org or other services.

Final point to note: it is possible to import users manually inside this service without going through the ORCID authentication, so profiles are not necessarily tied to an ORCID account. They will never get updated in that case (it's for imports of historical data).